### PR TITLE
Adopt the Scala Code of Conduct

### PR DESCRIPTION
**Pull request by rossabaker**
Typelevel is [switching to the Scala Code of Conduct](https://typelevel.org/blog/2019/05/01/typelevel-switches-to-scala-code-of-conduct.html) and encourages all member projects to adopt it.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+Everyone is expected to follow the [Scala Code of Conduct] when
+discussing the project on the available communication channels. If you
+are being harassed, please contact us immediately so that we can
+support you.
+
+## Moderation
+
+For any questions, concerns, or moderation requests please contact a
+member of the project.
+
```
Comment (rossabaker) [inline CODE_OF_CONDUCT.md]: If there are any project moderators, please append a commit to this PR here.  If not, the link below lists some Typelevel admins who are happy to assist.

**Comment by etorreborre** [comment]
Thanks @rossabaker I added my email address to the list of moderators as a separate commit.

**Comment by etorreborre** [comment]
By the way I noticed that the code of conduct link for cats-effect links to a missing page: https://typelevel.org/cats-effect/CODE_OF_CONDUCT.md

**Comment by rossabaker** [comment]
Thanks for the tip on cats-effect.  I'll go figure out what went wrong there.  (I suspect it's from copying something that works on GitHub.)

**Pull request merged**